src/asynk/async_queue.rs

```diff
@@ -141,6 +140,82 @@ where
     pool: Pool<PostgresConnectionManager<Tls>>,
 }
 
+pub struct AsyncQueueTest<'a> {
```

Comment:
👏 

let's add `#[cfg(test)]` because it will be used only for tests

Reply:
Done :D